src/views/RandomView.vue

```diff
@@ -17,20 +17,63 @@ import { ref } from "vue"
       return booster.value = newBooster;
       })
   }
+
+  const getNewDrink = () => {
+    fetch("https://www.thecocktaildb.com/api/json/v1/1/random.php")
+    .then(response => {
+      console.log(response)
+      return response.json()
+      })
+    .then(data => {
+      const ingredientsList = [];
+      for (let i = 1; i < 16; i++) {
+        if (data.drinks[0][`strIngredient${i}`]) {
+          const ingredient = data.drinks[0][`strMeasure${i}`] + " " + data.drinks[0][`strIngredient${i}`]
+          ingredientsList.push(ingredient)
+        }
+      }
+      const newBooster = {
+        title: data.drinks[0].strDrink,
+        preparation: data.drinks[0].strInstructions,
+        ingredients: ingredientsList,
+        id:Date.now(),
+        type:"drink",
+        isFavorited:false
+      }
+      return booster.value = newBooster;
+  })
+      // const newBooster = {
+      //   text: data.text,
```

Comment:
This commented out section could be removed eventually on a future refactor branch but no rush.